deploy/service.yaml

```diff
@@ -1,106 +0,0 @@
----
```

Comment:
Please do not delete this file because some automation still uses it.

Reply:
i have re-added the file